### ISSUE TITLE
Help please with new style Keele University

### ISSUE THREAD

**Issue by andyhubbert941** [citation-style-language/styles#6863]
Hi all

Very new to this so please be kind. I have the following style guide from Keele University for their School of Allied Health professionals. It's a pretty big group to support so I think fits the remit of a style. They also promote refworks... even though there is no style in there even for this!!

My issue is I'm simply not sure how to even start to create this style. States it's based on Vancouver (author-date) but the actual bibliography doesn't appear to be?

Any help with creating this would be absolutely amazing as I'm very reliant on Zotero at this point!

Thanks

[SAHP_REFERENCE_MANUAL_1.5b.docx](https://github.com/citation-style-language/styles/files/14024519/SAHP_REFERENCE_MANUAL_1.5b.docx)

**Comment by andyhubbert941** [comment]
My only other thought... is this just Harvard but they're being complicated?!

**Comment by adam3smith** [comment]
Yeah, this is a Harvard-type style. Looks roughly similar to some of the Springer styles we have, so that's one possible starting point, but also, we have the search by example tool in the editor for exactly this scenario:
https://editor.citationstyles.org/searchByExample/

**Comment by andyhubbert941** [comment]
Sorry yes - I should've used that. Found and edited a style so here is the file. Not sure how I contribute this back but here in case others need it (had to use a link as not supported filetype)

https://cloud.andyhubbert.co.uk/s/M43BLFrYBj4rqSo

**Comment by andyhubbert941** [comment]
sorry yews should've found that!! I've now edited one and created a pull request #6866
